src/main/java/survey/MultipleChoiceQuestion.java

```diff
@@ -15,17 +13,14 @@
 public class MultipleChoiceQuestion extends Question {
 
     private HashMap<String,Integer> options;
-    @Enumerated(EnumType.ORDINAL)
```

Comment:
what was this ordinal

Reply:
unimportant for when i was attempoting to add the enum, wasnt necessary so i removed from the other 2 question types but missed this one